Formula/minio-mc.rb

```diff
@@ -24,8 +24,18 @@ def install
     clipath.install Dir["*"]
 
     cd clipath do
-      system "go", "build", "-o", bin/"mc"
+      if build.head?
+        system "go", "build", "-o", buildpath/"mc"
```

Comment:
Why the change here to the simpler mechanism becoming a head-only thing? Did something break in current stable?